raiden/network/proxies/service_registry.py

```diff
@@ -140,16 +137,17 @@ def set_url(self, url: str) -> None:
             msg = "URL provided to service registry must be a valid HTTP(S) endpoint."
             raise BrokenPreconditionError(msg)
 
-        with log_transaction(log, "set_url", log_details):
```

Comment:
the context manager is not necessary anymore, logging is done by the `transaction` objects.